### PR TITLE
Add catalan translations

### PR DESCRIPTION
**Pull request by victormhg**
Added catalan translations to bundle

### CODE REVIEW

File: src/Resources/translations/ResetPasswordBundle.ca.xlf [hunk 1]
```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" version="1.2">
+    <file original="file.ext" source-language="en" datatype="plaintext">
```
Comment (bocharsky-bw) [inline src/Resources/translations/ResetPasswordBundle.ca.xlf:3]: Missing `target-language` attr, see any diffrent from `en` translation:

```suggestion
    <file original="file.ext" source-language="en" target-language="ca" datatype="plaintext">
```

Comment (victormhg) [reply src/Resources/translations/ResetPasswordBundle.ca.xlf]: I've added the target language attribute, nevertheless a lot of other language files hasn't this attribute.

Comment (victormhg) [reply src/Resources/translations/ResetPasswordBundle.ca.xlf]: May be not "a lot" but "some" ;-)

**Review by bocharsky-bw — APPROVED** [review]
Thank you for adding more translations!

**Review by jrushlow — APPROVED** [review]
Thank you @victormhg

**Pull request merged**